action.yaml

```diff
@@ -59,6 +59,8 @@ inputs:
   st2_packs:
     description: String. Comma separated list of packs to install. This flag does not work with a --python3 only pack. (default `"st2"`). If you modify this option, be sure to also include `st2` in the list.
     default: "st2"
+  st2_conf:
+    description: Relative path from Project Root to configuration file.
```

Comment:
maybe add a note that this will replace the [default version](https://github.com/bitovi/github-actions-deploy-stackstorm/blob/main/operations/deployment/ansible/vars/st2.yaml) of the file.